src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/VisualStudioWorkspaceProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Razor.Workspaces;
+
+namespace Microsoft.VisualStudio.LanguageServices.Razor;
+
+[Export(typeof(IWorkspaceProvider))]
+[method: ImportingConstructor]
+internal sealed class VisualStudioWorkspaceProvider(VisualStudioWorkspace workspace) : IWorkspaceProvider
+{
+    public Workspace GetWorkspace() => workspace;
+}
```

Comment:
⚔️ This 15line file is a perfect example of how a primary constructor can greatly cleanup boilerplate for simple code. Fight me!

Reply:
Had to argue against this, I'll admit :)

(Other than perhaps arguing in the form of "Why does this class and interface even exist, when consumers could just import `VisualStudioWorkspace`?)

Reply:
Because some consumers are in the language server, where `VisualStudioWorkspace` is not accessible.